soc/arm/nordic_nrf/nrf52/dts_fixup.h

```diff
@@ -19,13 +19,41 @@
 #define DT_GPIO_P0_DEV_NAME		DT_NORDIC_NRF_GPIO_GPIO_0_LABEL
 #define DT_GPIO_P1_DEV_NAME		DT_NORDIC_NRF_GPIO_GPIO_1_LABEL
 
-#define DT_I2C_0_NAME			DT_NORDIC_NRF_I2C_I2C_0_LABEL
-#define DT_I2C_1_NAME			DT_NORDIC_NRF_I2C_I2C_1_LABEL
+#if defined(DT_NORDIC_NRF_TWIM_I2C_0_LABEL)
+#define DT_I2C_0_NAME			DT_NORDIC_NRF_TWIM_I2C_0_LABEL
+#else
+#define DT_I2C_0_NAME			DT_NORDIC_NRF_TWI_I2C_0_LABEL
+#endif
+
+#if defined(DT_NORDIC_NRF_TWIM_I2C_1_LABEL)
+#define DT_I2C_1_NAME			DT_NORDIC_NRF_TWIM_I2C_1_LABEL
+#else
+#define DT_I2C_1_NAME			DT_NORDIC_NRF_TWI_I2C_1_LABEL
+#endif
 
+#if defined(DT_NORDIC_NRF_SPIM_SPI_0_LABEL)
+#define DT_SPI_0_NAME			DT_NORDIC_NRF_SPIM_SPI_0_LABEL
+#else
 #define DT_SPI_0_NAME			DT_NORDIC_NRF_SPI_SPI_0_LABEL
+#endif
+
+#if defined(DT_NORDIC_NRF_SPIM_SPI_1_LABEL)
+#define DT_SPI_1_NAME			DT_NORDIC_NRF_SPIM_SPI_1_LABEL
+#else
 #define DT_SPI_1_NAME			DT_NORDIC_NRF_SPI_SPI_1_LABEL
+#endif
+
+#if defined(DT_NORDIC_NRF_SPIM_SPI_2_LABEL)
+#define DT_SPI_2_NAME			DT_NORDIC_NRF_SPIM_SPI_2_LABEL
+#else
 #define DT_SPI_2_NAME			DT_NORDIC_NRF_SPI_SPI_2_LABEL
+#endif
+
+#if defined(DT_NORDIC_NRF_SPIM_SPI_3_LABEL)
+#define DT_SPI_3_NAME			DT_NORDIC_NRF_SPIM_SPI_3_LABEL
+#else
 #define DT_SPI_3_NAME			DT_NORDIC_NRF_SPI_SPI_3_LABEL
```

Comment:
There is no SPI3 in nRF52 Series, only SPIM3.

Reply:
Fixed